style.css

```diff
@@ -1,6 +1,22 @@
 body {
 	text-align: center;
-	background: #aaf;
+	background-image: url("ship_image_resize.jpg");
+	background-repeat: no-repeat;
+}
+
+h1{
+	text-align: left;
+	color: silver;
+
+	font-size: 30pt;
+	font-family: "Bahnschrift Light";
```

Comment:
I think this font is only available in Windows. It would be good to add some web safe fallback options: https://www.w3schools.com/cssref/css_websafe_fonts.asp